nucliadb/nucliadb/tasks/consumer.py

```diff
@@ -45,13 +45,17 @@ def __init__(
         stream: const.Streams,
         callback: TaskCallback,
         msg_type: MsgType,
+        max_concurrent_messages: Optional[int] = None,
```

Comment:
Doubt: `None` is the same as 1 right?

Reply:
None is the nats default, which is no limit on max_ack_pending

Reply:
We have to be careful with too many simultaneous tasks though

Reply:
agree! I've set a maximum of 10 for exports imports and defaulted to settings.max_ack_pending if not set (which right now is set to 1)